### PR TITLE
task(27156897): update more 6kyu tests

### PR DESCRIPTION
**Pull request by vmred**

### CODE REVIEW

File: katas/6kyu/Number Zoo Patrol/test_number_zoo_patrol.py [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+cases = [Case([2, 3, 4], 1), Case([1, 2, 3], 4), Case([1, 2, 4], 3)]
 
 
 class TestSolution:
-    def test_number_zoo_patrol(self):
-        assert_true(find_missing_number([2, 3, 4]), 1)
-        assert_true(find_missing_number([1, 2, 3]), 4)
-        assert_true(find_missing_number([1, 2, 4]), 3)
+    @pytest.mark.parametrize('test', cases, ids=[f'{test.test_data}' for test in cases])
+    def test_find_missing_number(self, test):
```
Comment (red-redalert) [inline katas/6kyu/Number Zoo Patrol/test_number_zoo_patrol.py:14]: should be `test_number_zoo_patrol`

File: katas/6kyu/Reverse or rotate/test_reverse_or_rotate.py [hunk 2]
```diff
@@ -2,12 +2,16 @@
...
+solution = importlib.import_module('katas.6kyu.Reverse or rotate.solution').revrot
+
+cases = [
+    Case(["3304991787281570455176064327690480265895", 8], '7199403307518278606715543276904402658958'),
+]
 
 
 class TestSolution:
-    @pytest.mark.xfail
-    @pytest.mark.not_competed
```
Comment (red-redalert) [inline katas/6kyu/Reverse or rotate/test_reverse_or_rotate.py:11]: missed marks

**Review by red-redalert — CHANGES_REQUESTED** [review]

File: katas/6kyu/Number Zoo Patrol/test_number_zoo_patrol.py (hunk 1 above)
Comment (vmred) [reply katas/6kyu/Number Zoo Patrol/test_number_zoo_patrol.py:14]: fixed

File: katas/6kyu/Reverse or rotate/test_reverse_or_rotate.py (hunk 2 above)
Comment (vmred) [reply katas/6kyu/Reverse or rotate/test_reverse_or_rotate.py:11]: done

**Review by red-redalert — DISMISSED** [review]

**Review by red-redalert — APPROVED** [review]

**Pull request merged**